godot-codegen/src/special_cases/special_cases.rs

```diff
@@ -135,9 +127,12 @@ pub fn is_godot_type_deleted(godot_ty: &str) -> bool {
         | "MovieWriterMJPEG"
         | "MovieWriterPNGWAV"
         | "ResourceFormatImporterSaver"
-        | "UniformSetCacheRD"
-
-        => true, _ => false
+        => true,
+        // Previously loaded lazily; in 4.2 it loads at the Scene level. See: https://github.com/godotengine/godot/pull/81305
+        | "ThemeDB" => cfg!(before_api = "4.2"),
+        // reintroduced in 4.3. See: https://github.com/godotengine/godot/pull/80214
+        | "UniformSetCacheRD" => cfg!(before_api = "4.3"),
+        _ => false
```

Comment:
Sorry for nitpicking, but I _explicitly_ asked to copy-paste because there are several capitalization/period problems :wink: 

The empty line before the comment is removed by rustfmt? If yes, that's fine, but please refer to my previous code block regarding the comment wording/formatting.